util/lib.py

```
@@ -342,41 +342,54 @@ def load_yaml(filepath):
     return data
 
 
-def robot_prepare_ontology(o_path, o_out_path, o_metrics_path, base_iris, make_base, robot_prefixes={}, robot_opts="-v"):
-    logging.info(f"Preparing {o_path} for dashboard.")
-    
-    callstring = ['robot', 'merge', '-i', o_path]
-    
+def robot_prepare_ontology(
```

Ruff

Missing return type annotation for public function `robot_prepare_ontology`

Add return type annotation: `None`

(ANN201)